juriscraper/opinions/united_states/federal_appellate/ca10.py

```diff
@@ -24,22 +38,43 @@ def _get_download_urls(self):
     def _get_case_dates(self):
         dates = []
         for date_string in self.html.xpath("//item/pubdate/text()"):
-            date_only = " ".join(date_string.split(" ")[1:4])
-            dates.append(
-                date.fromtimestamp(
-                    time.mktime(time.strptime(date_only, "%d %b %Y"))
+            try:
+                span_element = html.etree.fromstring(str(date_string))
+                date_string = span_element.xpath("/span/text()")[0]
+                date_only = " ".join(date_string.split(" ")[1:4])
+                dates.append(
+                    date.fromtimestamp(
+                        time.mktime(time.strptime(date_only, "%b %d %Y"))
+                    )
```

Comment:
Did you check/try the tool for this we have in date_utils? It should make it easier.

Reply:
Changed to `convert_date_string`